### ISSUE TITLE
Dark mode not working on dropdown menus

### ISSUE THREAD

**Issue by mikojimnz** [mikojimnz/DIG4104C#9]
**Describe the bug**
Dark mode not working on nav manus

**To Reproduce**
1. Go to 'Settings' > 'Language & Accessibility Preferences'
2. Enable 'Dark Mode'
3. Click on a dropdown menu

**Expected behavior**
Dropdown background should be dark in order to see inverted text.

**Screenshots**
<img width="416" alt="Screen Shot 2021-10-31 at 6 01 09 PM" src="https://user-images.githubusercontent.com/16627839/139602613-8d82294e-35bd-4072-8669-98dd67e8a634.png">
